test/fixtures/include-path/lib-alternate/vars.scss

```diff
@@ -0,0 +1 @@
+$color: orange;
```

Comment:
Sorry I didn't quite follow that. Do you mean put my fixtures in their own directory called "sass-path" or something?